index.js

```diff
@@ -44,8 +44,19 @@ const renderHelper = (tasks, options, level) => {
 	return output.join('\n');
 };
 
+const shouldSuspendUpdateRenderer = tasks => {
+	for (const task of tasks) {
+		if (task.shouldSuspendUpdateRenderer()) {
```

Comment:
Maybe use [`tasks.some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here to simplify the code?

Reply:
Done